dvc/command/experiments.py

```diff
@@ -155,7 +155,11 @@ def _sort_exp(experiments, sort_by, typ, reverse):
         ret = OrderedDict()
 
     def _sort(item):
-        _, exp = item
+        rev, exp = item
+        tip = exp.get("checkpoint_tip")
+        if tip and tip != rev:
```

Comment:
if `tip == rev`, this is tip of a checkpoint branch and we want to use the actual metric/param values from this experiment